routes/index.js

```diff
@@ -298,26 +379,25 @@ module.exports = function (passport) {
     if (!req.body) {
       return res.sendStatus(400);
     }
-    var BagdeID = req.params.badge;
-
-    dbAPI.validateUser(BagdeID).then(function (ret) {
-      console.log(ret.dataValues);
-      console.log("USER STATUS " + ret.status);
+    var badgeID = req.params.badge;
 
+    dbAPI.validateUser(badgeID).then(function (ret) {
       if (ret.status == "User" && ret.status != "Admin") {
-        dbAPI.modifyUser(BagdeID, { status: "Manager" }).then(function (result) {
+        dbAPI.modifyUser(badgeID, { status: "Manager" }).then(function (result) {
           if (result == undefined) {
-            console.log("Error");
+            flash("error", "Internal error: unable to promote " + badgeID);
```

Comment:
Shouldn't this be `req.flash()`?

Reply:
Whoops! Definitely. I'll fix that real quick.

Reply:
Done.